src/ingest_llm_as/api/repository.py

```
@@ -71,6 +81,45 @@ async def ingest_python_repository(
+        raw_record = RawIngestion(
+            ingestion_id=ingestion_id,
+            source_type="python-repo",
+            content_type="repository",
+            raw_payload=request.model_dump(mode="json"),
+            raw_metadata={
+                "repository_source": request.repository_source.value,
+                "source_path": request.source_path,
+                "max_files": request.max_files,
+                "max_file_size": request.max_file_size,
+                "include_patterns": request.include_patterns,
+                "exclude_patterns": request.exclude_patterns[:METADATA_LIST_TRUNCATION_LIMIT],  # First 5 for brevity
```

Ruff (0.14.11)

Undefined name `METADATA_LIST_TRUNCATION_LIMIT`

(F821)